src/Language/PureScript/Linter/Exhaustive.hs

```diff
@@ -54,7 +42,11 @@ data RedudancyError = Incomplete | Unknown
 -- |
 -- Qualifies a propername from a given qualified propername and a default module name
 --
-qualifyName :: a -> ModuleName -> Qualified a -> Qualified a
+qualifyName
+  :: (ProperName a)
+  -> ModuleName
+  -> Qualified (ProperName b)
```

Comment:
Apparently, yeah - sometimes a constructor name is used to qualify a type name or vice versa.